Sources/BraintreeApplePay/BTApplePayAnalytics.swift

```diff
@@ -1,6 +1,5 @@
 import Foundation
 
-// NEXT_MAJOR_VERSION: Move this class to an enum once the entire SDK is in Swift
```

Comment:
Removing this comment, ticket DTBTSDK-2772 exists for this task. Don't think we need a note in 1 of the classes since this will need to be updated in all of the modules.